### ISSUE TITLE
Remove leading slash

### ISSUE THREAD

**Issue by shirobachi** [harshil1712/n8n-nodes-n8n-api#4]
From baseUrl leading slash needs to be removed in case user will time 127.0.0.1/ isntead of 127.0.0.1

**Comment by harshil1712** [comment]
Good catch. Let's finish adding all the resources and then we can tackle this!

**Comment by shirobachi** [comment]
Also let's auto add http if not passed
